pop/service/service.go

```diff
@@ -91,9 +91,9 @@ type Service struct {
 	path string
 	data *saveData
 	// propagate final message
-	PropagateFinalize messaging.PropagationFunc
-	// propagate merge info
-	PropagateMerging messaging.PropagationFunc
+	propagateFinalize messaging.PropagationFunc
+	// propagate possible new descriptions
+	propagateDescription messaging.PropagationFunc
```

Comment:
I imagine this change would change the API

Reply:
Got me. But this should have been private anyway, nobody else should be calling the `propagateMerging` function that is not initialised anywhere!